src/tables.jl

```diff
@@ -24,6 +24,19 @@ getquery(r::Row) = getfield(r, :q)
 Tables.isrowtable(::Type{Query}) = true
 Tables.columnnames(q::Query) = q.names
 
+struct DBTable
+    name::String
+    schema::Union{Tables.Schema, Nothing}
+end
+
+DBTable(name::String) = DBTable(name, Tables.schema(""))
+
+DBTables = AbstractVector{DBTable}
+
+Tables.istable(::DBTables) = true
+Tables.rowaccess(::Type{<:DBTables}) = true
+Tables.rows(dbtbl::DBTables) = dbtbl
```

Comment:
```suggestion
Tables.isrowtable(::Type{DBTables}) = true
```
we can actually replace all 3 of these lines with this one